app/models/recommendation.rb

```diff
@@ -25,6 +25,52 @@ def validate(record)
 
   before_validation :set_pending_status
 
+  def self.create_by_id_and_email(recommender, params)
+    recommendee_ids = params["recommendee_ids"]
+    recommendee_emails = params["recommendee_emails"]
+    item_class = Kernel.const_get(params['item_type'])
```

Comment:
Check for errors here. What if I pass in `ConstThatDoesntExist` as the `item_type`?